### ISSUE TITLE
About Training

### ISSUE THREAD

**Issue by chanwo0kim** [hwjiang1510/GraspTTA#3]
Hello. I am a student studying Hand-Object mesh.

First of all, congratulations on your research success.

I have a few questions and I would appreciate it if you could answer them.

1. When will the learning code be released?
2. It seems that Penetration, Vertex, Translation, Pose, and KLD loss are used when learning CVAE in the paper. Could you please tell me what the weight was?

We will wait for your reply. thank you.

**Comment by hwjiang1510** [comment]
Hi, thanks for your attention to our work.
I am sorry that I am busy at the moment, I don't have much time to clean up the training code.
But of course, you can send me an email, I will give you a raw version.

**Comment by chanwo0kim** [comment]
Thank you for your kindness!

If you send me a code by email, I think it will be useful for studying!

I'll be waiting for your reply.

Thank you!


2022년 2월 11일 (금) 오전 6:09, Hanwen Jiang ***@***.***>님이 작성:

**Comment by hwjiang1510** [comment]
Sorry, could you tell me your email or directly email me?

**Comment by chanwo0kim** [comment]
Oh, I'm sorry I didn't leave an email by mistake.

My email address is sky9739a@gmail.com.

Thank you!

**Comment by abhinavkk** [comment]
Hope you are doing great. I was working on studying hand-object point clouds and came across your work. I was hoping that you could send me a raw version of the training code too. It will be very useful and I would appreciate your help.

**Comment by trannguyenle95** [comment]
@hwjiang1510 Can you also send me the raw version to tranlqd@gmail.com? Thanks a lot!
